src/ScheduleCommand.js

```diff
@@ -0,0 +1,22 @@
+import './Command'
+
+export class ScheduleCommand extends Command {
+
+	name = 'schedule:run'
+	description = 'Starts the schedule daemon and executes commands as scheduled.'
+
+	async run() {
+		this.info('Schedule daemon starting...')
+
+		const scheduled = []
+
+		for(var command of this.cli.commands) {
+			if(typeof(command.schedule) === 'function') {
```

Comment:
`typeof` doesn’t need to be a function, just `typeof command.schedule === 'function'` will suffice